source/adios2/toolkit/transport/file/FilePOSIX.cpp

```diff
@@ -2,13 +2,12 @@
  * Distributed under the OSI-approved Apache License, Version 2.0.  See
  * accompanying file Copyright.txt for details.
  *
- * FileDescriptor.cpp file I/O using POSIX I/O library
+ * FilePOSIX.cpp file I/O using POSIX I/O library
  *
- *  Created on: Oct 6, 2016
- *      Author: William F Godoy godoywf@ornl.gov
```

Comment:
Would be nice to  keep.

Reply:
If you look at the history of the file it went through multiple revisions and it has many authors. I think we should remove the header from all the files since there is a copyright file that is covering the entire repo and includes all the authors (you are included). I'll bring it up next Tuesday and if everyone agrees I can can create a PR.